src/shogun/base/init.cpp

```diff
@@ -39,6 +41,7 @@ namespace shogun
 	Version* sg_version=NULL;
 	CMath* sg_math=NULL;
 	CRandom* sg_rand=NULL;
+	CSignal* sg_signal = NULL;
```

Comment:
we could make this one a `std::unique_ptr<CSignal> sg_signal(nullptr)` since we are c++11 and not c++98